xarray/core/daskmanager.py

```diff
@@ -97,6 +97,26 @@ def reduction(
             keepdims=keepdims,
         )
 
+    def scan(
+        self,
+        func: Callable,
+        binop: Callable,
+        ident: float,
+        arr: T_ChunkedArray,
+        axis: int | None = None,
+        dtype: np.dtype | None = None,
+    ):
```

Comment:
```suggestion
    ) -> T_ChunkedArray:
```
Or should it be even more narrow here? `dask.array.Array`?

Reply:
Yep should have been dask array, thanks for catching that.